doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -157,31 +157,20 @@ the Bourne shell (sh)                ``. util/quickstart/setchplenv.sh``
 What's next?
 ------------
 
-=============================================== =========================
+=============================================== ==================================================
```

Comment:
This one could also be reduced to 80-columns.  The table itself can also almost be squeezed to 80 columns if the 5+ spaces between the columns are compressed out.  Some of the entries could be tightened up to help more (e.g., "environment variables" -> "environment vars", "file and directory" -> "file/dir" but the header in the first column is still problematic...  :(